docs/README-agent-install.md

```diff
@@ -601,11 +602,15 @@ systemctl start my_chef
 
 [Chef Provisioning](README-chef-provisioning.md) - Installing & Configuring chef-client using the Chef Provisioning tool
 
-## References
+## <a name="references">References
```

Comment:
Interesting.  The auto anchors don't work in MacDown, but they do on github.  I will remove it.